.yarnrc.yml

```diff
@@ -278,6 +278,9 @@ packageExtensions:
     peerDependencies:
       react: '*'
       redux: '*'
+  '@wordpress/rich-text@5.1.0':
```

Comment:
I would avoid adding this, and instead install rich-text@5.0.7. We'll be updating these packages in https://github.com/Automattic/wp-calypso/pull/60161, and we'll make sure all the peer dependency issues get resolved :)